boards/nxp/imx93_evk/imx93_evk_mimx9352_a55_ramconsole.dts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "imx93_evk-pinctrl.dtsi"
```

Comment:
Most of dts nodes are the same with imx93_evk_mimx9352_a55.dts, how about include imx93_evk_mimx9352_a55.dts directly? then only add updates in this file, it will be easy to maintain.

Reply:
As Jamie suggested the snippet, will try to convert to snippet, so just add overlay dts. Thanks!